documentation/modules/ova-prerequisites.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="ova-prerequisites_{context}"]
+= Open Virtual Applications (OVA) prerequisites
+
+The following prerequisites apply to Open Virtual Application (OVA) file migrations:
+
+* All OVA files are created by or compatible with VMware vSphere.
+* The OVA files are in a shared NFS share that has the following security settings: `rw,insecure`.
```

Comment:
After some tests, the insecure is not needed, only rw.

Reply:
@bkhizgiy did you check with `r` instead of `rw`?

Reply:
so after testing provider adding + migration seems that read only is enough (even the insecure redundent).
so it can be changed to `(ro)`

Reply:
@RichardHoch probably missed but we can change `rw,insecure` to `ro`